site/content/en/docs/manual/advanced/serverless-tutorial.md

```diff
@@ -215,61 +213,6 @@ CVAT will run the serverless function on every frame of the task and submit
-21.07.12 16:48:59.071            nuctl.platform (I) Waiting for function to be ready {"timeout": 60}
-21.07.12 16:49:00.437                     nuctl (I) Function deploy complete {"functionName": "tf-matterport-mask-rcnn", "httpPort": 49155}
-```
-
-</details>
-
-Now you should be able to annotate objects using segmentation masks.
-
-![Mask RCNN results](/images/mask_rcnn_results.jpg)
```

Comment:
Please check that this image is not used anywhere else and also delete it.

Reply:
I don't think that it is not used anywhere as I searched the whole website when I was removing mentions of mask rcnn I searched the whole website

Reply:
Ok, great.
Could deploy some other available gpu function and change the tutorial to use that function?

Reply:
> Ok, great. Could deploy some other available gpu function and change the tutorial to use that function?

I am still getting familiar with codebase, I would do it by tomorrow